### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by stephanie1997-kek**

### CODE REVIEW

**Comment by stephanie1997-kek** [comment]
ready

**Comment by stephanie1997-kek** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,57 @@
+source "https://rubygems.org"
+
+git_source(:github) do |repo_name|
+  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
+  "https://github.com/#{repo_name}.git"
+end
+
+# Bundle edge Rails instead: gem "rails", github: "rails/rails"
```
Comment (vanvtt-0952) [inline Gemfile]: plz, remove all comment code

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
+    <%= csrf_meta_tags %>
+
+    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```
Comment (vanvtt-0952) [inline app/views/layouts/application.html.erb]: plz, use `double-quotes`
ex: ` <%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track": "reload" %>`

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
```
Comment (vanvtt-0952) [inline app/views/layouts/application.html.erb]: use `double-quotes`

File: config/database.yml [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# SQLite version 3.x
```
Comment (vanvtt-0952) [inline config/database.yml]: this file will not be up to GIt, please
`cp database.yml database.yml.example`
`git rm --cached config/database.yml`
and in .gitignore add
`config/database.yml`

**Comment by stephanie1997-kek** [comment]
ready

File: Gemfile [hunk 5]
```diff
@@ -0,0 +1,32 @@
+source "https://rubygems.org"
+
+git_source(:github) do |repo_name|
+  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
+  "https://github.com/#{repo_name}.git"
+end
+
+gem "rails", "~> 5.1.7"
+gem "sqlite3"
```
Comment (ngocvt-0484) [inline Gemfile]: please remove it
and add gem "mysql2", run bundle install and start server

Comment (stephanie1997-kek) [reply Gemfile]: How to solve this error?
image

**Comment by stephanie1997-kek** [comment]
ready

File: Gemfile (hunk 5 above)
Comment (stephanie1997-kek) [reply Gemfile]: done

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**